### PR TITLE
[fr] MULTI_ADJ

### PR DESCRIPTION
**Pull request by remymartinDev**
the rule does not work for the moment , more a save than a PR

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -13645,6 +13645,24 @@ though, and has since been slightly modified)-->
...
+                    <token postag="J.*" postag_regexp="yes"/>
+                <unify-ignore>
+                    <token min="0" max="1">,</token>
+                    <token>et|ni</token>
+                </unify-ignore>
+                    <token postag="J.*" postag_regexp="yes"/>
+                </unify>
+            </pattern>
+            <message>L'adjectif "\4" ne semble pas accordé</message>
+            <suggestion><match no="4" postag="J*" postag_regexp="yes" postag_replace="$1"></match></suggestion>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:13662]: `J*` → `J.*`

The `$1` dans `postag_replace` doesn't know what to refer to

<match></match> is an "empty string" → <match/>

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -13645,6 +13645,24 @@ though, and has since been slightly modified)-->
...
     <category id="CAT_GRAMMAIRE" name="Grammaire">
+        <rule id="MULTI_ADJ" name="accord entre plusieurs adjectifs">
+            <pattern>
+                <unify negate="yes">
+                    <feature id="number"/>
+                    <feature id="gender"/>
+                    <token postag="J.*" postag_regexp="yes"/>
+                <unify-ignore>
+                    <token min="0" max="1">,</token>
+                    <token>et|ni</token>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:13656]: regexp="yes"

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 3]
```diff
@@ -13645,6 +13645,24 @@ though, and has since been slightly modified)-->
...
     </category>
     <category id="CAT_GRAMMAIRE" name="Grammaire">
+        <rule id="MULTI_ADJ" name="accord entre plusieurs adjectifs">
+            <pattern>
+                <unify negate="yes">
+                    <feature id="number"/>
+                    <feature id="gender"/>
+                    <token postag="J.*" postag_regexp="yes"/>
+                <unify-ignore>
+                    <token min="0" max="1">,</token>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:13655]: max="1" is the default, you can leave it out

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 4]
```diff
@@ -13645,6 +13645,24 @@ though, and has since been slightly modified)-->
         </rulegroup>
     </category>
     <category id="CAT_GRAMMAIRE" name="Grammaire">
+        <rule id="MULTI_ADJ" name="accord entre plusieurs adjectifs">
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:13648]: default="temp_off"

**Comment by jaumeortola** [comment]
Added manually.

**Pull request closed without merging**